code/modules/mob/living/simple_animal/hostile/terror_spiders/empress.dm

```diff
@@ -0,0 +1,263 @@
+		to_chat(src, "Cancelled.")
+		return
+	// T1
+	if (eggtype == "red - assault")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/red,numlings,1)
+	else if (eggtype == "gray - ambush")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/gray,numlings,1)
+	else if (eggtype == "green - nurse")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/green,numlings,1)
+	// T2
+	else if (eggtype == "black - poison")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/black,numlings,1)
+	else if (eggtype == "purple - guard")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/purple,numlings,0)
+	// T3
+	else if (eggtype == "PRINCE")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/prince,numlings,1)
+	else if (eggtype == "MOTHER")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/mother,numlings,1)
+	else if (eggtype == "QUEEN")
+		DoLayTerrorEggs(/mob/living/simple_animal/hostile/poison/terror_spider/queen,numlings,1)
+	// Unrecognized
+	else
+		to_chat(src, "Unrecognized egg type.")
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/HiveSense()
+	set name = "Hive Sense"
+	set category = "Spider"
+	set desc = "Sense the spiders at your command."
+	DoHiveSense()
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/HiveCommand()
+	set name = "Hive Command"
+	set category = "Spider"
+	set desc = "Instruct your brood on proper behavior."
+	DialogHiveCommand()
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/EMPShockwave()
+	set name = "EMP Shockwave"
+	set category = "Spider"
+	set desc = "Emit a wide-area emp pulse, frying almost all electronics in a huge radius."
+	empulse(src.loc,10,25)
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/EmpressScreech()
+	set name = "Empress Screech"
+	set category = "Spider"
+	set desc = "Emit horrendusly loud screech which breaks lights and cameras in a massive radius. Good for making a spider nest in a pinch."
+	for(var/obj/machinery/light/L in range(14,src))
+		L.on = 1
+		L.broken()
+	for(var/obj/machinery/camera/C in range(14,src))
+		if (C.status)
+			C.toggle_cam(src,0)
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/empress/verb/EmpressMassHallucinate()
+	set name = "Mass Hallucinate"
+	set category = "Spider"
+	set desc = "Causes widespread, terrifying hallucinations amongst many crew as you assault their minds."
+	var/numaffected = 0
+	for(var/mob/living/carbon/human/H in player_list)
+		if (H.z != src.z)
+			continue
+		if (H.health < 1)
+			// nothing
+		else if (prob(50))
+			// nothing
+		else if (prob(50))
+			// weak
+			H.hallucination = max(300, H.hallucination)
+			to_chat(H,"<span class='userdanger'> Your head hurts! </span>")
+			numaffected++
+		else
+			// strong
+			H.hallucination = max(600, H.hallucination)
```

Comment:
this could just be in the conditional above
```
if(prob(50))
    var/new_hallucination = 300
    if(prob(50))
        new_hallucination = 600
    H.hallucination = max(new_hallucination, H.hallucination)
    etc...